### PR TITLE
[Core] Update README to point Wiki

### PR DESCRIPTION
**Pull request by loumalouomega**
**Description**
Added links to wiki

**Changelog**
- Added links to wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,28 @@ Some main modules are:
...
+* [Running an example from GiD](https://github.com/KratosMultiphysics/Kratos/wiki/Running-an-example-from-GiD)
+* [Kratos input files and I/O](https://github.com/KratosMultiphysics/Kratos/wiki/Kratos-input-files-and-IO)
+* [Data management](https://github.com/KratosMultiphysics/Kratos/wiki/Data-management)
+* [Solving strategies](https://github.com/KratosMultiphysics/Kratos/wiki/Solving-strategies)
+* [Manipulating solution values](https://github.com/KratosMultiphysics/Kratos/wiki/Manipulating-solution-values)
+* [Multiphysics](https://github.com/KratosMultiphysics/Kratos/wiki/Multiphysics-example)
+
+## More documentation
+
+[Wiki](https://github.com/KratosMultiphysics/Kratos/wiki)
```
Comment (rubenzorrilla) [inline README.md]:
```suggestion
## More documentation
[Wiki](https://github.com/KratosMultiphysics/Kratos/wiki)

```

**Comment by loumalouomega** [comment]
Done @rubenzorrilla

**Review by rubenzorrilla — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -40,6 +40,28 @@ Some main modules are:
 - [Metis](applications/MetisApplication/README.md)
 - [Meshing](applications/MeshingApplication/README.md)
 
+# Documentation
+Here you can find the basic documentation of the project:
+## Getting Started
+* Getting Kratos (Last compiled Release)
+    * [Kratos for Linux](https://github.com/KratosMultiphysics/Kratos/wiki/Getting-Kratos-Binaries-for-Linux)
+    * [Kratos for Windows](https://github.com/KratosMultiphysics/Kratos/wiki/Getting-Kratos-Binaries-for-Windows)
```
Comment (rubenzorrilla) [inline README.md:48]: I'd point to INSTALL.md in here

File: README.md [hunk 3]
```diff
@@ -40,6 +40,28 @@ Some main modules are:
...
+    <!-- * [Kratos for Mac](https://github.com/KratosMultiphysics/Kratos/wiki/MacOS-install) -->
+    * [Kratos for GiD](https://github.com/KratosMultiphysics/Kratos/wiki/Getting-Kratos-binaries-(via-GiD))
+* Compiling Kratos
+    * [See INSTALL.md](https://github.com/KratosMultiphysics/Kratos/blob/master/INSTALL.md)
```
Comment (loumalouomega) [inline README.md:52]: It is here!

File: README.md (hunk 2 above)
Comment (loumalouomega) [reply README.md:48]: This is precompiled version

File: README.md (hunk 3 above)
Comment (loumalouomega) [reply README.md:52]: This is to compile @rubenzorrilla

File: README.md (hunk 2 above)
Comment (rubenzorrilla) [reply README.md:48]: I know. But I'd point to how to install too. In any case this is just a temporary solution (I assume that once we have the final documentation platform we would only point there).

Comment (loumalouomega) [reply README.md:48]: But the install is in the line 52

File: README.md (hunk 3 above)
Comment (loumalouomega) [reply README.md:52]: Here

Comment (rubenzorrilla) [reply README.md:52]: :facepalm: OK. I didn't notice.

**Pull request merged**